src/commands/data/import/resume.ts

```diff
@@ -0,0 +1,193 @@
+/*
+ * Copyright (c) 2023, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import { SfCommand, Flags } from '@salesforce/sf-plugins-core';
+import { Messages } from '@salesforce/core';
+import { MultiStageOutput } from '@oclif/multi-stage-output';
+import terminalLink from 'terminal-link';
+import { JobInfoV2 } from '@jsforce/jsforce-node/lib/api/bulk2.js';
+import { BulkImportRequestCache } from '../../../bulkDataRequestCache.js';
+
+Messages.importMessagesDirectoryFromMetaUrl(import.meta.url);
+const messages = Messages.loadMessages('@salesforce/plugin-data', 'data.import.resume');
+
+export type DataImportResumeResult = {
+  jobId: string;
+  processedRecords?: number;
+  successfulRecords?: number;
+  failedRecords?: number;
+};
+
+export default class DataImportResume extends SfCommand<DataImportResumeResult> {
+  public static readonly summary = messages.getMessage('summary');
+  public static readonly description = messages.getMessage('description');
+  public static readonly examples = messages.getMessages('examples');
+
+  public static readonly flags = {
+    'use-most-recent': Flags.boolean({
+      summary: messages.getMessage('flags.use-most-recent.summary'),
+      exactlyOne: ['job-id'],
+    }),
+    'job-id': Flags.salesforceId({
+      summary: messages.getMessage('flags.job-id.summary'),
+      char: 'i',
+      length: 18,
+      startsWith: '750',
+      exactlyOne: ['use-most-recent'],
+    }),
+    wait: Flags.duration({
+      char: 'w',
+      unit: 'minutes',
+      summary: messages.getMessage('flags.wait.summary'),
+      defaultValue: 5,
```

Comment:
`data import resume` has a default wait time of 5 minutes.
This is on purpose so that if you don't specify `--wait` you still have a good chance the job finishes on time.

`data upsert resume` and `data delete resume` have `--wait` with `0` as a default, so on a first run without `--wait` you have to re-run them again with a bigger timeout if the job is still running.